snuba/clickhouse/columns.py

```diff
@@ -41,26 +52,26 @@ def __init__(self, base_name: Optional[str], name: str, type: ColumnType) -> Non
         self.flattened = (
             "{}.{}".format(self.base_name, self.name) if self.base_name else self.name
         )
-        self.escaped = escape_identifier(self.flattened)
+        self.escaped: str = escape_identifier(self.flattened)
```

Comment:
I think an assertion here would be safer than assuming that `escape_identifier` doesn't return the `None` specified in the signature.

Reply:
I am ok adding the assertion. I did this way in order not to hide the type error in the escape_identifier signature.

Reply:
Ok, that makes sense too — either way works though I think if this does return `None` it'd be a problem?

Reply:
I'll add the assertion AND the type, so we do not loose the type error and still we avoid null.